docs/src/joint/api/dia/Paper/prototype/defineGradient.html

```diff
@@ -1,27 +1,61 @@
 <pre class="docs-method-signature"><code>paper.defineGradient(gradientDefinition)</code></pre>
 
-<p>Define an SVG gradient for later reuse within the paper. The method returns the gradient id and the <code>gradientDefinition</code> must have the following form:</p>
+<p>Define an SVG gradient for later reuse within the paper. The method returns the gradient id and the <code>gradientDefinition</code> must be an object with the following properties:</p>
 
-<pre><code>{
-    type: &lt;type of gradient&gt;,
-    stops: &lt;stop colors&gt;,
-    attrs: &lt;additional attributes&gt;
-}</code></pre>
+<table>
+    <tr>
+        <th>Name</th>
+        <th>Type</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+        <td>type</td>
+        <td><code>'linearGradient'</code> | <code>'radialGradient'</code></td>
+        <td>The <a target="_blank" href=https://developer.mozilla.org/en-US/docs/Web/SVG/Element/linearGradient>linearGradient</a> or <a target="_blank" href=https://developer.mozilla.org/en-US/docs/Web/SVG/Element/radialGradient>radialGradient</a> type of the gradient.</td>
+    </tr>
+    <tr>
+        <td>id</td>
+        <td>string</td>
+        <td> <i>(optional)</i> - A unique identifier of the gradient. The <i>id</i> is generated if none provided.</td>
```

Comment:
There is an extra space here. The same on:
line `15` in `docs/src/joint/api/dia/Paper/prototype/defineMarker.html ` +
line `14` in `docs/src/joint/api/dia/Paper/prototype/definePattern.html`